app/src/main/res/drawable/ic_close.xml

```diff
@@ -1,6 +1,6 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="40dp"
-    android:height="40dp"
+    android:width="24dp"
```

Comment:
Make this a dimen?

Reply:
Dimens in vectors are usually not used. Because the final size is calculated by the view that uses this vector. And it used to be not stable to use dimens in vectors but I guess the support of it has changed already and it shouldn't be a problem. 

Let's review this later.